core/src/main/java/edu/ucr/cs/riple/core/metadata/field/FieldInitializationAnalysis.java

```diff
@@ -24,10 +24,10 @@ public class FieldInitializationAnalysis extends MetaData<FieldInitializationNod
    * Constructs an {@link FieldInitializationAnalysis} instance. After this call, all serialized
    * information from NullAway has been processed.
    *
-   * @param path Path to field initialization info coming from NullAway.
+   * @param config Annotator config.
    */
-  public FieldInitializationAnalysis(Path path) {
-    super(path);
+  public FieldInitializationAnalysis(Config config) {
+    super(config, config.target.dir.resolve("field_init.tsv"));
```

Comment:
Could we extract `"field_init.tsv"` into a global constant, like we do below for `MethodDeclarationTree`:

```
    super(config, config.target.dir.resolve(Serializer.METHOD_INFO_FILE_NAME));
```

Reply:
Actually `"field_init.tsv"` is not an output of `TypeAnnotatorScanner` and it might not be preferred to declare the file name in its `Serializer`. But we should definitely declare the file as a global constant. I extracted it into the user class. caff1d3c011c485fdb178886e55465f29297c22e